commands/addons/create.js

```diff
@@ -2,12 +2,13 @@
 
 const cli = require('heroku-cli-util')
 const co = require('co')
+let waitForAddonProvisioning = require('../../lib/addons_wait')
 
 function parseConfig (args) {
   let config = {}
   while (args.length > 0) {
     let key = args.shift()
-    if (!key.startsWith('--')) throw new Error(`Unexpected argument ${key}`)
+    if (!key.startsWith('--')) { throw new Error(`Unexpected argument ${key}`) }
```

Comment:
ooooh I see, this is for things like `--fork`/`--follow` that are non-standard. Might be useful to add a comment to that effect – it's not easily discernible from this wall of code.